Patch by Zarafa <http://www.zarafa.com/> - adds required include headers to compile with g++-4.3 and up.

--- libvmime-0.7.1/src/messaging/imap/IMAPMessage.cpp			2005-05-27 21:31:44.000000000 +0200
+++ libvmime-0.7.1/src/messaging/imap/IMAPMessage.cpp.gcc-4.3-support	2008-07-07 15:41:38.034166062 +0200
@@ -26,7 +26,7 @@
 
 #include <sstream>
 #include <iterator>
-
+#include <typeinfo>
 
 namespace vmime {
 namespace messaging {
--- libvmime-0.7.1/src/platforms/posix/posixSocket.cpp			2005-03-18 22:26:49.000000000 +0100
+++ libvmime-0.7.1/src/platforms/posix/posixSocket.cpp.gcc-4.3-support	2008-07-07 15:42:16.700025085 +0200
@@ -19,6 +19,7 @@
 
 #include "vmime/platforms/posix/posixSocket.hpp"
 
+#include <string.h>
 #include <unistd.h>
 #include <sys/socket.h>
 #include <arpa/inet.h>
--- libvmime-0.7.1/src/utility/md5.cpp					2005-03-18 22:26:48.000000000 +0100
+++ libvmime-0.7.1/src/utility/md5.cpp.gcc-4.3-support			2008-07-07 15:41:38.034166062 +0200
@@ -45,7 +45,7 @@
 //     documentation and/or software.
 
 #include "vmime/utility/md5.hpp"
-
+#include <string.h>
 
 namespace vmime {
 namespace utility {
--- libvmime-0.7.1/vmime/utility/filteredStream.hpp			2005-06-16 21:19:12.000000000 +0200
+++ libvmime-0.7.1/vmime/utility/filteredStream.hpp.gcc-4.3-support	2008-07-07 15:41:38.034166062 +0200
@@ -22,7 +22,7 @@
 
 
 #include "vmime/utility/stream.hpp"
-
+#include <algorithm>
 
 namespace vmime {
 namespace utility {
